src/main/groovy/grails/plugin/sentry/SentryConfig.groovy

```diff
@@ -112,6 +116,7 @@ class SentryConfig {
     boolean logHttpRequest = false
     boolean disableMDCInsertingServletFilter = false
     boolean springSecurityUser = false
+    boolean sanitizeStackTrace = false
```

Comment:
it might be more useful when it is enabled by default, WDYT?

Reply:
Yes we can do that, if you think that it's okey to change the default behavior of the plugin